structs.inc

```diff
@@ -315,18 +324,19 @@ MACRO dstruct ; struct_type, instance_name[, ...]
 				ENDC
 				; Add padding as necessary after the provided initializer
 				; (possibly all of it, especially for RAM use)
-				IF {STRUCT_FIELD_SIZE} < @ - \2_{{STRUCT_FIELD_NAME}}
-					fail STRFMT("Initializer for %s is %u bytes, expected %u at most", "\2_{{STRUCT_FIELD_NAME}}", @ - \2_{{STRUCT_FIELD_NAME}}, {STRUCT_FIELD_SIZE})
+				IF {STRUCT_FIELD_SIZE} < @ - {STRUCT_FIELD_START}
+					fail STRFMT("Initializer for %s is %u bytes, expected %u at most", "{STRUCT_FIELD_START}", @ - {STRUCT_FIELD_START}, {STRUCT_FIELD_SIZE})
 				ENDC
-				ds {STRUCT_FIELD_SIZE} - (@ - \2_{{STRUCT_FIELD_NAME}})
+				ds {STRUCT_FIELD_SIZE} - (@ - {STRUCT_FIELD_START})
 			ENDC
 
 			purge_nth_field_info
+			purge STRUCT_FIELD_START
```

Comment:
```suggestion
			purge_nth_field_info, STRUCT_FIELD_START
```

Reply:
`purge_nth_field_info` doesn't work like that.

Reply:
I, uh. Somehow misread the line as `purge nth_field_info`, lol